web_external/js/views/StudyResults/StudyResultsView.js

```diff
@@ -470,20 +287,20 @@ isic.views.StudyResultsView = isic.View.extend({
     events: {
         // Update image visibility when image preview tab is activated
         'shown.bs.tab #isic-study-results-image-preview-tab': function (event) {
-            this.localFeaturesImageView.setVisible(false);
+            this.localFeaturesView.setVisible(false);
```

Comment:
It's confusing that the visibility for the element `#isic-study-results-local-features-tab-content`' is controlled both through the Bootstrap tab mechanism and in code.